src/ChildProcessUtilities.js

```diff
@@ -33,7 +36,8 @@ export default class ChildProcessUtilities {
 
   static execSync(command, opts) {
     const mergedOpts = objectAssign({
-      encoding: "utf8"
+      encoding: "utf8",
+      maxBuffer: 500 * 1024
```

Comment:
Would be nice to have this pulled out as a module-level constant so it's not duplicated.

```javascript
const MAX_BUFFER = 500 * 1024;
```

That way when we (inevitably) need to increase it again we won't accidentally do one an miss the other.